staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -935,6 +953,37 @@ func (g *genValidations) emitValidationForChild(c *generator.Context, thisChild
 				case types.Struct:
 					// Call the type's validation function.
 					g.emitCallToOtherTypeFunc(c, fld.node, bufsw)
+					for _, subchild := range fld.inner {
+						if len(subchild.name) == 0 {
+							klog.Fatalf("missing child name for field in %v", thisNode)
+						}
+						if len(subchild.jsonName) == 0 {
+							klog.Fatalf("missing child JSON name for field %v.%s", thisNode, subchild.name)
+
```

Comment:
wasted line - gotta conserve the vertical space!

Reply:
done!